### PR TITLE
TILA-336 reservation unit filtering

### PR DESCRIPTION
**Pull request by siren**

### CODE REVIEW

File: src/common/util.ts [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
   const now = new Date();
   return isAfter(now, parseISO(startDate)) && isBefore(now, parseISO(endDate));
 };
 
+const isPast = (endDate: string): boolean => {
+  const now = new Date();
+  return isAfter(now, parseISO(endDate));
+};
+
+export const uiState = (
```
Comment (kieferbonk) [inline src/common/util.ts:14]: This could use a bit more descriptive name.

File: src/search/SearchForm.tsx [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+import { mapOptions, OptionType, getSelectedOption } from '../common/util';
 
-interface Props {
-  // only text search is now implemented!
-  onSearch: (text: string) => void;
-}
-interface OptionType {
-  label: string;
-}
+export type Criteria = {
```
Comment (kieferbonk) [inline src/search/SearchForm.tsx:10]: Maybe not needed to be exported, or if so, belongs to to the type definition file?

**Review by kieferbonk — APPROVED** [review]

File: src/search/SearchForm.tsx (hunk 2 above)
Comment (siren) [reply src/search/SearchForm.tsx]: fixed

File: src/common/util.ts (hunk 1 above)
Comment (siren) [reply src/common/util.ts]: true, renamed

**Pull request merged**